dist/stat_data.py

```diff
@@ -327,6 +332,22 @@ def __init__(self, name, desc, flags=''):
     CapacityStat('fsync_all_fh_total', 'background fsync file handles considered'),
     CapacityStat('fsync_all_time', 'background fsync time (msecs)', 'no_clear,no_scale'),
 
+    ##########################################
+    # Chunk cache statistics
+    ##########################################
+    ChunkCacheStat('chunk_cache_bytes', 'total bytes'),
+    ChunkCacheStat('chunk_cache_chunks', 'total chunks'),
+    ChunkCacheStat('chunk_cache_chunks_evicted', 'evicted chunks'),
+    ChunkCacheStat('chunk_cache_chunks_invalidated', 'removed chunks on becoming invalid'),
```

Comment:
Typically in a stat description, we put the verb second. So for both of these I suggest `chunks evicted` and `chunks removed on becoming invalid`. (For comparison, there are stats for `pages deleted` and `pages visited`, etc).